### DISCUSSION TITLE
error, fat_free: invalid cluster chain ??

### DISCUSSION THREAD

**Discussion by teredactle** [gtxaspec/wz_mini_hacks#607] [Q&A]
Hi all, I'm seeing this error in the logs, over and over.
Latest mini-hacks with WC3.
Used a utility for format the 64gb SD card to fat32 in windows, which I think may be the issue here.

Can someone chime in and/or recommend a windows utility I can use to format the 64gb card to FAT32 and not have this error?

[  475.394352] FAT-fs (mmcblk0p1): error, fat_free: invalid cluster chain (i_pos 3639852)
[  475.896150] FAT-fs (mmcblk0p1): error, fat_free: invalid cluster chain (i_pos 3639852)
[  476.405998] FAT-fs (mmcblk0p1): error, fat_free: invalid cluster chain (i_pos 3639852)
[  476.906480] FAT-fs (mmcblk0p1): error, fat_free: invalid cluster chain (i_pos 3639852)
[  477.412555] fat_get_cluster: 156 callbacks suppressed
[  477.424913] FAT-fs (mmcblk0p1): error, fat_get_cluster: invalid cluster chain (i_pos 3639852)
[  477.434906] FAT-fs (mmcblk0p1): error, fat_free: invalid cluster chain (i_pos 3639852)

no SD card error
SD read only error found
[ 79.570268] FAT-fs (mmcblk0p1): Filesystem has been set read-only
no file system error
no SD card (tf_prepare) error
no SD card health fail error

TY

**Comment by archandanime** [comment]
Maybe your SD Card partition has corrupted data, the only solution I can think of is to enable fsck on the SD Card at boot:
```
ENABLE_FSCK_ON_BOOT="true"
```
